tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
```

Comment:
I'm answering because if we have installed our package (python setup.py install) to dev venv it could lead to magical bugs because of to copies of it in PYTHONPATH.

Again - i never did it that way - correct me if i wrong.